include/circt/Dialect/OM/OMOps.td

```diff
@@ -75,6 +87,15 @@ def ObjectOp : OMOp<"object",
     ClassType:$result
   );
 
+  let builders = [
+    OpBuilder<(ins "om::ClassOp":$classOp, "::mlir::ValueRange":$params),
+    [{ return build($_builder, $_state,
+                   om::ClassType::get($_builder.getContext(),
+                   mlir::FlatSymbolRefAttr::get(classOp)),
```

Comment:
Can you please indent this line, so it is more clear that this is part of the call to `ClassType::get`? Or, define the method in the C++ file, so it gets formatted, etc.

Reply:
Moved the def to C++.